src/main/webapp/js/script.js

```diff
@@ -0,0 +1,36 @@
+let globalCount = 0;
```

Comment:
ditto. I think these changes are unrelated. Please remove the changes to these files.